Tasks/SonarQubePostTest/PRCA/Orchestrator.ps1

```diff
@@ -1,18 +1,113 @@
-
 . $PSScriptRoot/ReportProcessor.ps1
+. $PSScriptRoot/IssuesProcessor.ps1
 
 function HandleCodeAnalysisReporting
 {	
     Write-Verbose "HandleCodeAnalysisReporting..."
     if (IsPrBuild)
     {   
-        Write-Host "Fetching code analysis issues and posting them to the PR..." 
+        Write-Host "Fetching code analysis issues and posting them to the PR..."        
         $newIssues = FetchAnnotatedNewIssues                
         
-        if ($newIssues.Length -eq 0)
-        {            
+        if ( ($newIssues -eq $null) -or ($newIssues.Length -eq 0))
+        {
             Write-Host "The SonarQube analysis did not find any new issues."
-            return            
+            return
         }
+
+        LoadClientOMAssemblies 
+        PostIssuesToCodeReview $newIssues   
     }	
 }
+
+#region Private 
+
+function PostIssuesToCodeReview
```

Comment:
Not much point in testing this since all it does is ask for objects and read env variables.